src/index.ts

```diff
@@ -326,6 +332,28 @@ export async function install(
     };
     const packageBundle = await rollup.rollup(inputOptions);
     await packageBundle.write(outputOptions);
+
+    if (include && nomodule) {
+      // Resolve `/web_modules/` relative to package.json rather than system root
+      function renameImportPlugin() {
+        return {
+          name: 'rename-import-plugin',
+          resolveId(source) {
+            if (source.startsWith(path.sep)) {
+              return {id: source.replace(path.sep, path.join(cwd, path.sep))};
+            }
+            return null; // null means don’t rename
+          },
+        };
+      }
+      const noModuleBundle = await rollup.rollup({
+        input: glob.sync(include, {ignore: exclude})[0], // Note: this isn’t a good substitute for an entry file
+        plugins: [...inputOptions.plugins, renameImportPlugin()],
+      });
+      await noModuleBundle.write({file: nomodule, format: 'iife'});
+      spinner.info(`${chalk.bold('snowpack')} created nomodule bundle: ${nomodule}`);
```

Comment:
Also curious: what are your thoughts about adding info for this nomodule bundle? This is what it looks like:

<img width="1002" alt="Screen Shot 2020-01-11 at 3 57 42 PM" src="https://user-images.githubusercontent.com/1369770/72211602-2ab7ab80-348b-11ea-8223-b609b98f6e21.png">

Reply:
Ping @FredKSchott 👆 wanted to get your feedback on this messaging (or whether or not to include it)

Reply:
The most important part! I would love to see it a little more human readable, something like:

```
snowpack bundled your application for legacy browsers: web_modules/app.nomodule.js
snowpack installed: ...
```

Also:
- the `web_modules/app.nomodule.js` should be some sort of color (blue?) to match the others
- if you can easily add timing to the legacy bundle, that would be great

Reply:
Actually, maybe more important than time info would be size info? Feel free to go with whatever feels best to you

Reply:
I’d rather do time for consistency in this PR with snowpack, if that’s all right. But you have me thinking about size now…

(rambly thoughts): while it would be neat to see size for `nomodule`, IMO it wouldn’t mean as much without comparing to snowpack’s output (e.g. “using your ES module bundle would save `250 KB` (`2s` on slow connections)”). But even then, size is a little dishonest with ES modules because that paints the picture that an ES module app == 1 large webpack bundle.

So all that to say… this is tough! I’d like to think about output a little more—something closer to what webpack did in v3 with bundle size warnings. But whatever the ES module version is of that: letting users know the impact of snowpack. Would love to hear your thoughts here!

Reply:
lets keep time for now, then. 

Maybe there's a `--profile` or `--with-stats` mode that we can add that adds size info for nomodule PLUS all installed packages. Spun off an issue here to prioritize here: https://github.com/pikapkg/snowpack/issues/163